packages/component-dashboard/client/components/DashboardList.js

```diff
@@ -2,9 +2,9 @@ import React from 'react'
 import styled from 'styled-components'
 import { Box } from '@rebass/grid'
 import { th } from '@pubsweet/ui-toolkit'
+import Paragraph from 'ui/atoms/Paragraph'
 
-import DashboardListItem from 'ui/molecules/DashboardListItem'
-import Paragraph from '../atoms/Paragraph'
+import DashboardListItem from './DashboardListItem'
```

Comment:
I didn't realise that we moved things out of our ui folder ... was that intentional?

Reply:
I believe anything unique to the package is moved with it.

Reply:
Yeah, think that's right.... especially given it has the word "Dashboard" in its name :laughing: